src/main/scala/nl/knaw/dans/api/sword2/package.scala

```diff
@@ -43,6 +43,8 @@ package object sword2 {
 
   case class BagStoreSettings(baseDir: String, baseUrl: String)
 
+  case class PropertiesResources(bagId: String, filePaths: List[String])
```

Comment:
I looked into changing the type of the parameters from String to URL, but I am wondering if we should do it.  These values are input for org.swordapp.server.ResourcePart, where they are given as String (even though representing an IRI).  We store these values in Properties, also as Strings, and thus have to convert them when reading from or writing to Properties file. To what avail actually?  Wouldn’t it be more logical just to use the same type as in ResourcePart or in Properties, namely String?